src/EmailUsers.py

```diff
@@ -0,0 +1,28 @@
+import yagmail
+
+# yagmail.SMTP() without parameters will attempt to locate a .yagmail file in the home directory
+# that contains the gmail username. Should be used in the future to avoid having
+# credentials placed in code
+
+yag = yagmail.SMTP('bugtrackerp@gmail.com', 'D@nkC0d3rs')
```

Comment:
The credentials may need to be retrieved from the config file -- I don't think its safe for this software to have to rely on Gmail credentials to be able to send an email. We should rather be using something like a token

Reply:
This should also be wrapped in a function (like the class constructor may be a good choice) - which then would mean the rest of the functions should also fall under this class